README.md

```diff
@@ -92,6 +120,22 @@ A minimal example of using an existing configuration.nix:
   };
 }
 ```
+</details>
+
+Instead of using `darwin-installer`, you can just run `darwin-rebuild switch` to install nix-darwin. As `darwin-rebuild` won't be installed in your `PATH` yet, you can use the following command:
+
+```bash
+nix run nix-darwin#darwin-rebuild -- switch --flake ~/.config/nix-darwin#Johns-MacBook
+```
+
+After installing, you can run `darwin-rebuild` directly:
+
+```bash
+darwin-rebuild switch --flake ~/.config/nix-darwin#Johns-MacBook
+```
+
+> NOTE: The `hostname(1)` of your system will be used to decide
+> which darwin configuration is applied if it's not specified explicitly in the flake ref.
```

Comment:
Maybe say `hostname -s` instead:

```console
emily@yuyuko ~> hostname 
yuyuko.emily.moe
emily@yuyuko ~> hostname -s
yuyuko
```

The latter is used by `darwin-rebuild`.

Reply:
FYI there's is #676 which would change the behaviour slightly.

Reply:
Yeah, I'm agnostic to which we use; I might slightly prefer the current solution as I've had Bonjour clashes before. Mostly just want to document the correct one whichever it is.

Reply:
Updated :+1: